features/math-sum-precise.yml.dist

```diff
@@ -6,7 +6,5 @@ status:
   support:
     firefox: "137"
     firefox_android: "137"
-    safari: "18.4"
-    safari_ios: "18.4"
```

Comment:
This was actually behind a flag: https://github.com/mdn/browser-compat-data/pull/27105